llvm/test/CodeGen/SystemZ/fp-conv-08.ll

```diff
@@ -2,6 +2,15 @@
 ;
 ; RUN: llc < %s -mtriple=s390x-linux-gnu -mcpu=z10 | FileCheck %s
 
+; Test i64->f16. For z10, this results in just a single a libcall.
+define half @f0(i64 %i) {
+; CHECK-LABEL: f0:
+; CHECK: brasl %r14, __floatundihf@PLT
```

Comment:
This libcall doesn't actually exist in our library, as far as I can see.  If we want to use it, we need to ensure that it is actually being built in both libgcc and compiler-rt.   Or else, we go via f32 like for the other conversions - that might be the more logical approach.